benchmarks/run.js

```diff
@@ -1,17 +1,21 @@
-var fixtures = require('./fixtures');
-var local = require('../');
-var dedupe = require('../dedupe');
-var localPackage = require('../package.json');
+import { readFile } from 'fs/promises'
```

Comment:
This deep `fs/promises` path is only available since Node.js v14.0.0, which is incompatible with the declared Node.js support in the package `engines.node` field.

https://nodejs.org/api/fs.html#fs_promises_api

Reply:
Since we're not shipping this code to end users and only running it on our CI it seems fine to use this.